bot/cogs/wolfram.py

```diff
@@ -0,0 +1,68 @@
+import discord
+from discord.ext import commands
+from urllib import parse
+from io import BytesIO
+import aiohttp
+import os
+
+
+APPID = os.environ.get("WOLFRAM_APPID")
+QUERY = "http://api.wolframalpha.com/v2/{request}?{data}"
+
+class Wolfram(commands.Cog):
```

Comment:
there should be 2 blank lines above a class definition.

Reply:
by this do you mean that i define `APPID` in settings.py, and then import it on the cog?